NuGet.config

```diff
@@ -25,6 +31,12 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-efcore -->
+    <add key="darc-int-dotnet-efcore-73adbdc-1" value="true" />
```

Comment:
@AndriySvyryd do you know if that is expected?

Reply:
Pretty sure it won't break anything but it's weird enough for me to notice it. 6 new feeds.